### PR TITLE
add the support of cardinality for servicebus

### PR DESCRIPTION
**Pull request by amamounelsayed**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/functions/annotation/ServiceBusQueueTrigger.java [hunk 1]
```diff
@@ -92,4 +92,12 @@
...
    */
   boolean isSessionsEnabled() default false;
+
+  /**
+   * Cardinality of the trigger input.
+   * Choose 'One' if the input is a single message or 'Many' if the input is an array of messages.
+   * 'ONE' is the default if unspecified
+   * @return An {@link Cardinality} value representing the Cardinality
+   */
+  Cardinality cardinality() default Cardinality.ONE;
```
Comment (TsuyoshiUshio) [inline src/main/java/com/microsoft/azure/functions/annotation/ServiceBusQueueTrigger.java:102]: I believe `dataType()` will be required for cardinality. 
https://github.com/Azure/azure-functions-java-library/pull/116/files
https://github.com/Azure/azure-functions-java-library/pull/112

File: src/main/java/com/microsoft/azure/functions/annotation/ServiceBusTopicTrigger.java [hunk 2]
```diff
@@ -101,4 +101,12 @@
...
    */
   boolean isSessionsEnabled() default false;
+
+  /**
+   * Cardinality of the trigger input.
+   * Choose 'One' if the input is a single message or 'Many' if the input is an array of messages.
+   * 'ONE' is the default if unspecified
+   * @return An {@link Cardinality} value representing the Cardinality
+   */
+  Cardinality cardinality() default Cardinality.ONE;
```
Comment (TsuyoshiUshio) [inline src/main/java/com/microsoft/azure/functions/annotation/ServiceBusTopicTrigger.java:111]: same as above.

**Review by TsuyoshiUshio — CHANGES_REQUESTED** [review]
Please add `dataType()` for cardinality.  The dataType is the type of the array. we can choose three types according to the Functions Host. Refer to KafkaTrigger PR or EventHubs PR. :)

File: src/main/java/com/microsoft/azure/functions/annotation/ServiceBusQueueTrigger.java (hunk 1 above)
Comment (amamounelsayed) [reply src/main/java/com/microsoft/azure/functions/annotation/ServiceBusQueueTrigger.java:102]: I agree the datetype is already added before

**Review by TsuyoshiUshio — APPROVED** [review]
LGTM. You can add explanation about the dataTyp() when you use cardinality, however, It's ok as is as well. :) Good job.

**Pull request merged**